content/en/blog/2019/announcing-istio-client-go/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Announcing Istio client-go"
+description: "Using Istio clien-go to gain programmatic access to Istio resources."
+publishdate: 2019-11-12
+attribution: Neeraj Poddar (Aspen Mesh)
+keywords: [client-go,tools,crd]
+target_release: 1.4
+---
+
+We are pleased to announce the official release of Istio
+[client go](https://github.com/istio/client-go) repository which enables users
+to gain programmatic access to Istio APIs in a Kubernetes environment. The
+generated Kubernetes informers and client set in this repository makes it easy
+for users to create controllers and perform Create, Read, Update and Delete
+(CRUD) operations for all Istio Customer Resource Definitions (CRDs).
+
+This was a highly requested functionality by many Istio users, as is evident
+from the feature requests on the clients generated by [Aspen Mesh
+Istio client go](https://github.com/aspenmesh/istio-client-go) and the [Knative
+project](https://github.com/knative/pkg/tree/a805b647f3f2aa6cec2f0490680377e5854775d9/apis/istio).
```

Comment:
Can we use "master" here?

Reply:
We can, I know knative is going to remove this package soon, so I didn't want to knowingly add a link that will be removed soon.

I'm fine either way, your call.